src/components/dapp-staking-v2/my-staking/MyDapps.vue

```diff
@@ -5,23 +5,25 @@
         <table id="my-table">
           <thead>
             <tr>
-              <th>dApps</th>
-              <th>Staked Amount</th>
-              <th>Total Earned</th>
-              <th>Manage</th>
+              <th>{{ $t('myDapps.dapps') }}</th>
+              <th>{{ $t('myDapps.stakedAmount') }}</th>
+              <th>{{ $t('myDapps.totalEarned') }}</th>
+              <th>{{ $t('myDapps.manage') }}</th>
             </tr>
           </thead>
           <tbody>
             <tr v-for="t in items" :key="t.id">
               <td>{{ t.name }}</td>
-              <td>{{ t.stakedAmount.toLocaleString() }} ASTR</td>
-              <td>{{ t.totalEarned.toLocaleString() }} ASTR</td>
+              <td><TokenBalance :balance="t.stakedAmount" symbol="ASTR" /></td>
```

Comment:
1. ` symbol="ASTR" `: You can get the native token symbol by `  const { nativeTokenSymbol } = useNetworkInfo();`
2. `<TokenBalance />` -> `<token-balance />`